policy/interfaces.go

```diff
@@ -24,6 +24,9 @@ type RuntimeReader interface {
 	// Name returns the process name of the Runtime.
 	Name() string
 
+	// NSPath returns the process name of the Runtime.
+	NSPath() string
+
```

Comment:
That doesn't seem to match the comment. NSPath returns the process name of the Runtime ? Where is this used?

Reply:
Will update the comments.

Two reasons:
1) it is used here
https://github.com/aporeto-inc/enforcerd/pull/545/files#diff-f46d3978f0ef10ba2d41f0994134fd57R419

2) All the parameters can be read from that interface. Why not NSPath ? Seems like an ommition